spring-boot/pom.xml

```diff
@@ -5,16 +5,16 @@
 
   <groupId>com.pingidentity.samples</groupId>
   <artifactId>user-registration-sample</artifactId>
-  <version>0.0.4-SNAPSHOT</version>
+  <version>0.0.5-SNAPSHOT</version>
 
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.1.6.RELEASE</version>
+    <version>2.2.4.RELEASE</version>
```

Comment:
If this version works properly, then u can use `spring-boot.version` property here

Reply:
Nope, it doesn't work. Just removed `spring-boot.version` property as it is not used anywhere.

Reply:
I meant we have a property `spring-boot.version` that u can use here